src/Adapter/Shop/Context.php

```diff
@@ -159,13 +159,12 @@ public function setShopGroupContext($id)
     }
 
     /**
-     * Update Multishop context for only one shop group.
+     * Update Multishop context to global context
      *
-     * @param int $id Shop id to set in the current context
      */
-    public function setAllContext($id)
+    public function setAllContext()
```

Comment:
To avoid break change please deprecate $id param, or set a default value and  ignore it.